### PR TITLE
Indicate latest heroic version in bug report template

### PR DESCRIPTION
**Pull request by arielj**
This PR adds a small change in the PR template to let users know which one is the latest version of Heroic, in case someone tries to report a bug in an older version.

Also moved the version to a dropdown so it's easier to pick between `Latest Stable` `Latest Beta` and `main branch from GitHub`.

This is how it looks:
image

Putting this here as an idea, I understand if there's a concern that this can get outdated and you prefer not to use this.

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [ ] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

**Review by redromnon — APPROVED** [review]
Looks cool :+1:

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -45,14 +45,26 @@ body:
     attributes:
       label: Screenshots
       description: If applicable, add screenshots to help explain your problem.
+  - type: markdown
+    attributes:
+      value: |-
+        ## Latest stable version of Heroic is 2.5.2. Please, update Heroic before reporting a bug if you have an older version.
```
Comment (redromnon) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:51]: One will need to update this line every time a new stable and beta version gets released.

Comment (flavioislima) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:51]: That's a good point. Although not a blocker, just one more thing to prepare when doing a new release, ahahaha

**Review by flavioislima — APPROVED** [review]
Looks good 👍🏽

**Pull request merged**